src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -1475,7 +1476,8 @@ public static boolean isDownloadPopupRequired(DatasetVersion datasetVersion) {
         }
         // 1. License and Terms of Use:
         if (datasetVersion.getTermsOfUseAndAccess() != null) {
-            if (!TermsOfUseAndAccess.License.CC0.equals(datasetVersion.getTermsOfUseAndAccess().getLicense())
+            License license = datasetVersion.getTermsOfUseAndAccess().getLicense();
+            if ( license == null || !license.isDefault()
```

Comment:
~UI question - is CC0 special? I.e. if the default is fairly restrictive, do we need these popups to show? Or will users understand that they get the site default. (Not a code issue per se - looks like the code works to only popup if you're not using the default - just a question about whether that's what should happen.)

Reply:
~yes. It's really a preference question/backward compatibility question - Dataverse currently doesn't warn you about CC0. The current PR, in which the default is still CC0 works the same way. If someone changes the default to something else, the question of whether they should be warned with a pop-up is an open question/one that the PR currently answers with 'no, we don't warn people when the data is available under the default for the site'. That seems good enough so I guess I'd suggest any change should be a new issue/PR.

Reply:
Sound fine to address this in new PR, if necessary. I'll just post some feedback we got via Twitter about the surprise of a CC0 license:

"While I'm very sympathetic to it, it really bugs me that OCUL's Dataverse still defaults a submission to Public Domain, and it's not obvious that it does it unless you're really looking."

That's from https://github.com/IQSS/dataverse/issues/1990#issuecomment-351063151